unix/prebuild.sh

```diff
@@ -1439,6 +1442,11 @@ libraries_platformcpu += libx86avx2fma3.a
 libx86avx2fma3_a_SOURCES =  `echo $files_x86avx2fma3`
 libx86avx2fma3_a_CXXFLAGS = \$(CXXFLAGS) -mavx2 -mfma
 endif
+if BUILD_x86avx512
+libraries_platformcpu += libx86avx512.a
+libx86avx512_a_SOURCES =  `echo $files_x86avx512`
+libx86avx512_a_CXXFLAGS = \$(CXXFLAGS) -mavx512f
```

Comment:
The avx512noise object uses avx2 intrinsics for vpermpd and native fma3 so the libx86avx512.a CXXFLAGS should include the fma flag. 

Consequently, this line should be: 
~~~
linelibx86avx512_a_CXXFLAGS = \$(CXXFLAGS) -mfma -mavx512f
~~~